README.md

```diff
@@ -58,3 +58,188 @@ select * where {
     <urn:a> ?p ?o  .
 }
 ```
+
+# Secured Kafka Sink with FailedRecordProducer Configuration
+
+This guide provides instructions on how to start a secured Kafka Sink while configuring
```

Comment:
This has to be re-worded imo to explain exactly what we configure and how. Please, avoid mentioning name of classes as the users of the Kafka SINK should not care about what's the name of the class that does the DLQ sending.